common/saber_base.h

```diff
@@ -34,6 +34,8 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(NEWFONT)			\
     DEFINE_EFFECT(LOW_BATTERY)			\
     DEFINE_EFFECT(POWERSAVE)                    \
+    DEFINE_EFFECT(BATTERY)                      \
```

Comment:
LOW_BATTERY is "automated" it would be triggered when lowbatt.wav is triggered.  BATTERY is "on demand" to give you an action to show the battery level effect at any time, in my prop file holding Aux and pressing power will show the style that's associated with BATTERY to see your current battery level.

Reply:
Maybe we can call it BATTERY_LEVEL instead?

Reply:
That works for me.